moncli/types.py

```diff
@@ -248,6 +253,28 @@ def _cast(self, value):
     def _export(self, value):
         return str(value)
 
```

Comment:
More of a style issue.  Please be sure that there are two lines between each class.